gnovm/pkg/gnolang/op_exec.go

```diff
@@ -433,6 +433,7 @@ EXEC_SWITCH:
 	}
 	switch cs := s.(type) {
 	case *AssignStmt:
+		m.recordCoverage(cs)
```

Comment:
What is the overhead of something like this, for VM ops?

Reply:
This should have condition check also. I forgot